packages/utxo-lib/tests/compose.test.ts

```diff
@@ -5,13 +5,31 @@ import { verifyTxBytes } from './compose.utils';
 import { composeTxFixture } from './__fixtures__/compose';
 import { fixturesCrossCheck } from './__fixtures__/compose.crosscheck';
 
+import { getRandomInt, arrayShuffle } from '@trezor/utils';
+
+jest.mock('@trezor/utils');
+
+(arrayShuffle as jest.Mock).mockImplementation(jest.requireActual('@trezor/utils').arrayShuffle);
```

Comment:
Nit: this would be more conservative approach. A combination of your new code & your previous code - instead of mocking whole import and restoring only `arrayShuffle`, this restores whole import, but only `getRandomInt` is mocked. Writing it this way makes it clear that we _only need to_ mock the getRandomInt.

Pick whatever you like better :slightly_smiling_face: 

```suggestion
jest.mock('@trezor/utils', () => ({
    ...jest.requireActual('@trezor/utils'),
    getRandomInt: jest.fn(),
}));
```

Reply:
Good point, improved!